Pipfile

```diff
@@ -5,8 +5,11 @@ name = "pypi"
 
 [packages]
 watson-developer-cloud = "*"
+tensorflow = "*"
+keras = "*"
 
 [dev-packages]
+"flake8" = "*"
 
 [requires]
-python_version = "3.7"
+python_version = "3.6"
```

Comment:
I don't mind using 3.6 but you'll have to remove the data classes stuff in the ibm script. Dataclasses were added in 3.7